DeleteButton.js

```diff
@@ -0,0 +1,19 @@
+export class DeleteButton {
+    constructor(iD, userContainer) {
+        this.url = 'https://jsonplaceholder.typicode.com/users'
+        this.iD = iD + 1;
+        console.log(this.iD)
+        const deleteButton = document.createElement("button");
+        deleteButton.classList.add('delete-button');
+        deleteButton.innerHTML = "DELETE"
+        userContainer.append(deleteButton);
+        deleteButton.addEventListener("click", this.deleteUser)
```

Comment:
create a separate funciton for the button initialization

Reply:
done